packages/builder/src/builder.js

```diff
@@ -44,6 +44,11 @@ export default class Builder {
       restart: null
     }
 
+    this.supportedExtensions = ['vue', 'js']
```

Comment:
There are common use cases with joined version of this array. We may somehow refactor it